specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/securitydomain.json

```diff
@@ -0,0 +1,330 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "KeyVaultClient",
+    "description": "The key vault client performs cryptographic key operations and vault operations against the Key Vault service.",
+    "version": "7.2-preview"
```

Comment:
You need the folder to match this string, it's why you don't get avocado CI:
> The API version of the swagger is inconsistent with its file path.

Path should be `specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/securitydomain.json`